src/product.html

```diff
@@ -0,0 +1,253 @@
+          Get the best out of your company <br> with our service
+        </p>
+      </div>
+
+      <div class="flex flex-col box-border bg-slate-100 rounded-tr-2xl mr-20">
+         <div class="grid grid-cols-3 p-12 gap-4 pt-24">
+        <div class="flex flex-col gap-4">
+          <div class="max-w-2">
+            <img src="../Images/Icon (00).svg" alt="">
+          </div>
+          
+          <h1 class="font-bold">
+            Security and privacy
+          </h1>
+          <p>
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor.
+          </p>
+        </div>
+
+        <div class="flex flex-col gap-4">
+          <div class="max-w-2">
+            <img src="../Images/Icon (11).svg" alt="">
+          </div>
+          
+          <h1 class="font-bold">
+            Security and privacy
+          </h1>
+          <p>
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor.
+          </p>
+        </div>
+
+        <div class="flex flex-col gap-4">
+          <div class="max-w-2">
+            <img src="../Images/Icon (22).svg" alt="">
+          </div>
+          
+          <h1 class="font-bold">
+            Security and privacy
+          </h1>
+          <p>
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor.
+          </p>
+        </div>
+
+      </div>
+
+      <div class="grid grid-cols-3 p-12 gap-4">
+        <div class="flex flex-col gap-4">
+          <div class="max-w-2">
+            <img src="../Images/Icon (3).svg" alt="">
+          </div>
+          
+          <h1 class="font-bold">
+            Security and privacy
+          </h1>
+          <p>
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor.
+          </p>
+        </div>
+
+        <div class="flex flex-col gap-4 mb-12">
+          <div class="max-w-2">
+            <img src="../Images/Icon (4).svg" alt="">
+          </div>
+          
+          <h1 class="font-bold">
+            Security and privacy
+          </h1>
+          <p>
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor.
+          </p>
+        </div>
+
+        <div class="flex flex-col gap-4">
+          <div class="max-w-2">
+            <img src="../Images/Icon (5).svg" alt="">
+          </div>
+          
+          <h1 class="font-bold">
+            Security and privacy
+          </h1>
+          <p>
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor.
+          </p>
+        </div>
+
+      </div>
+      </div>
+
+    </div>
+
+    <div class="grid grid-cols-2 p-12">
+      <div class="flex flex-col gap-[20px] items-start py-[100px] px-[10%]">
+        <div
+          class="font-[700] text-transparent bg-clip-text bg-gradient-to-r from-[#373FFF] to-[#4ecef8]"
```

Comment:
font-bold here , not need of weight specification

Reply:
`#373FFF` and `#4ecef8` add them in config

Reply:
This is another way to use tailwind i don't think it is a problem if it is supported